tools/hardware.yml

```diff
@@ -46,8 +46,10 @@ devices:
       # GICRedistributor
       - index: 1
         kernel: GICR_PPTR
-        # Assume 8 cores max
-        kernel_size: 0x100000
+        # Assume 12 cores max
+        # Changing this requires changing the GICR_SIZE constant
+        # in gic_v3.c as well.
+        kernel_size: 0x180000
```

Comment:
Why 12 and not 16? I wonder if it's not time to make GICR_SIZE a platform define, with the default being 8.

Reply:
Just don't want to be excessive, considering the existing boards only use 8 cores.

Reply:
But both untyped regions as hardware peripheral regions are usually power-of-two, so I don't see any downside to upping it to 16 instead of 12..

Reply:
I double checked `devices_gen.h`. Seems like the kernel devices are always mapped in as 4K pages, and with the GICR region specifically, it is not `userAvailable` so wouldn't be created as untyped right?